ecosystem/sep-0001.md

```diff
@@ -116,9 +116,10 @@ These fields go in the `stellar.toml` `[[CURRENCIES]]` list, one set of fields f
 
 Field | Requirements | Description
 ------|--------------|------------
-code | string (<= 12 char) | Token code
+code | string (<= 12 char) | Asset code. Required for assets that are issued using a Stellar account as opposed to a contract.
+issuer | `G...` string | Asset issuer Stellar public key. Required for assets that are issued using a Stellar account as opposed to a contract.
+contract | `C...` string | Contract ID of the asset contract. The asset must be compatible with the [SEP-41 Token Interface](sep-0041.md) to be defined here.
```

Comment:
```suggestion
code | string (<= 12 char) | Asset code. Omitted for custom contract tokens not associated with Classic assets.
issuer | `G...` string | Asset issuer Stellar public key. Omitted for custom contract tokens not associated with Classic assets.
contract | `C...` string | Contract ID of the [SEP-41](sep-0041.md)-compatible asset contract. Omitted for Classic assets not associated with contract tokens.
```

Most assets (probably > 90%) will be circulating in both Classic and Soroban environments. Also, the issuer that has an existing Classic asset may advertise a **custom** contract for Soroban instead of the SAC for this particular asset. To my mind, the wording I proposed better reflects this general idea.

Reply:
Worth noting that every Stellar asset has a contract, even if it hasn't yet been deployed, anyone can deploy it without any special authorisation. So an issuer can deploy a custom contract that they wish folks to use for their token, but if their token is already a Stellar asset they can't prevent someone else deploying the Stellar asset contract for their asset.

For example, if you're using the latest soroban-cli, you should be able to run the following command for any code and issuer, where the contract hasn't been deployed yet:

```
soroban contract asset deploy --asset [code]:[issuer] --network ... --source ...
```

Reply:
@leighmcculloch Yes, I know, we discussed this specific case here: https://discord.com/channels/897514728459468821/1201751319451553853/1202058781320826940

Reply:
Ah yes, thanks for the link!